examples/v4/bounding-box-leaflet-filter.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Bounding box filter dataview example</title>
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.2.0/dist/leaflet.css" />
+  <script src="https://unpkg.com/leaflet@1.2.0/dist/leaflet.js"></script>
+  <!--Include carto styles  -->
+  <link rel="stylesheet" href="../../dist/themes/css/cartodb.css" />
+  <!-- Include cartodb.js Library -->
+  <script src="../../dist/cartodb.uncompressed.js"></script>
+  <style>
+    #map {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+    }
+    #widgetC {
+      position: absolute;
+      top: 80px;
+      right: 0;
+      padding: 20px;
+      font-size: 2em;
+      background-color: rgba(255, 255, 255, 0.8);
+      z-index: 1000;
+    }
+    #widgetF {
+      position: absolute;
+      top: 0;
+      right: 0;
+      padding: 20px;
+      font-size: 2em;
+      background-color: rgba(255, 255, 255, 0.8);
+      z-index: 1000;
+    }
+  </style>
+</head>
+
+<body>
+  <div id="widgetF"></div>
+  <div id="widgetC"></div>
+  <div id="map"></div>
+  <script>
+    (async () => {
+      var map = L.map('map').setView([42.431234, -8.643616], 5);
+      L.tileLayer('https://cartodb-basemaps-{s}.global.ssl.fastly.net/rastertiles/voyager_nolabels/{z}/{x}/{y}.png').addTo(map);
+
+      // Create the Client
+      var client = new carto.Client({
+        apiKey: '84fdbd587e4a942510270a48e843b4c1baa11e18',
+        serverUrl: 'https://{user}.carto.com:443',
```

Comment:
This won't work soon. Could you please replace it by `'https://cartojs-test.carto.com:443'`?